### DISCUSSION TITLE
Correct database backup procedure when running inside VM with Veeam Backups

### DISCUSSION THREAD

**Discussion by SageBilt** [nextcloud/all-in-one#2047] [Questions] [answered]
So I have AIO running really nicely in a VM running Ubuntu 22 server. We are doing backups to a dedicated server using Veeam Backup & Replication. I'm assuming that this method of backing up will not preserve the Postgre database state at the time of the snap-shot is taken of the VM? What would be the correct method to preserve the database data for a clean restore in case of disaster recovery?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2037
